src/vue/backend/src/templates/AgentsSettings.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-    <idx-block tag="fieldset" className="agents-settings form-content">
+    <idx-block
+        tag="fieldset"
+        :className="templateClass"
+        :formDisabled="formDisabled">
```

Comment:
Where is `:formDisabled` from? I don't see it in the `idx-block` option
*Repeat for all instances